lib/chef/provider/package/rubygems.rb

```diff
@@ -401,7 +401,7 @@ def initialize(new_resource, run_context=nil)
         end
 
         def is_omnibus?
-          if RbConfig::CONFIG['bindir'] =~ %r!/opt/(opscode|chef)/embedded/bin!
+          if RbConfig::CONFIG['bindir'] =~ %r!/opt/(opscode|chef|chefdk)/embedded/bin!
```

Comment:
Updated in https://github.com/jfly/chef/commit/d27a5f8257eed6383f33acd0ab2f07655a69da19.